### PR TITLE
[tests-only][full-ci]removing usage of a stepdefination inside another stepdefination in WebDavLockingContext

### PR DESCRIPTION
**Pull request by ghost**
## Description

removing the use of a stepdefinition inside another stepdefinition in `WebDavLockingContext.php`

## Related Issue

- Related issue: https://github.com/owncloud/ocis/issues/8199

## How Has This Been Tested?

- test environment:
- test case 1:locally
- test case 2:ci

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/bootstrap/WebDavLockingContext.php [hunk 1]
```diff
@@ -883,6 +909,17 @@ public function publicUploadFileSendingLockTokenOfPublic(
...
+	}
+
+	/**
+	 * @param integer $count
+	 * @param string $file
+	 * @param string $user
+	 *
+	 * @return void
+	 */
+	public function checkCountOfLockFiles(int $count, string $file, string $user) : void {
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/WebDavLockingContext.php:922]: remove this function and use the code wherever required. seems few code

File: tests/acceptance/features/bootstrap/WebDavLockingContext.php [hunk 2]
```diff
@@ -831,19 +831,44 @@ public function publicUploadFileSendingLockTokenOfUser(
...
+	/**
+	 * @param string $filename
+	 * @param string $content
+	 * @param string $itemToUseLockOf
+	 * @param string $lockOwner
+	 * @param string $publicWebDAVAPIVersion
+	 *
+	 * @return ResponseInterface
+	 */
+	public function sendingLockTockenOfPublicUploadFileOfUser(
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/WebDavLockingContext.php:853]:
```suggestion
	public function publicUploadWithUserLockTocken(
```

File: tests/acceptance/features/bootstrap/WebDavLockingContext.php [hunk 3]
```diff
@@ -831,19 +837,44 @@ public function publicUploadFileSendingLockTokenOfUser(
...
+	/**
+	 * @param string $filename
+	 * @param string $content
+	 * @param string $itemToUseLockOf
+	 * @param string $lockOwner
+	 * @param string $publicWebDAVAPIVersion
+	 *
+	 * @return ResponseInterface
+	 */
+	public function publicUploadWithUserLockTocken(
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/WebDavLockingContext.php:859]:
```suggestion
	public function publicUploadWithUserLockToken(
```

File: tests/acceptance/features/bootstrap/WebDavLockingContext.php [hunk 4]
```diff
@@ -831,19 +837,44 @@ public function publicUploadFileSendingLockTokenOfUser(
...
+	 *
+	 * @return ResponseInterface
+	 */
+	public function publicUploadWithUserLockToken(
+		string $filename,
+		string $content,
+		string $itemToUseLockOf,
+		string $lockOwner,
+		string $publicWebDAVAPIVersion
+	) : ResponseInterface {
```
Comment (saw-jan) [inline tests/acceptance/features/bootstrap/WebDavLockingContext.php:865]:
```suggestion
	): ResponseInterface {
```

**Review by saw-jan — APPROVED** [review]
:+1:

**Review by SagarGi — APPROVED** [review]

**Pull request merged**